README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
+- Recursion: Recursion is when you have a method that calls itself. All recursive methods can be written iteratively.
+- Recursive Case: The initial case where there isn't a known number.
```

Comment:
Hmm... this doesn't make sense to me.

I'd say the recursive case is the case in which the method calls itself.